core/src/main/resources/jenkins/model/Messages.properties

```diff
@@ -76,3 +76,5 @@ ItemCategory.Uncategorized.DisplayName=Uncategorized
 ItemCategory.Uncategorized.Description=Item types that have not yet been categorized by their plugin maintainer.
 ItemCategory.StandaloneProjects.DisplayName=Standalone Projects
 ItemCategory.StandaloneProjects.Description=Create projects with a self-contained configuration and history. These projects can be at the top-level or grouped within folders.
+ItemCategory.NestedProjects.DisplayName=Nested Projects
```

Comment:
NIT: so this can confuse me  as a user. A Maven Project is a "Nested Project" if you have parent modules and many sub modules.
Technically Maven Projects are not in folders (even on disk) - but that seems like a distinction that some users will not understand.